build_tools/cmake/iree_bytecode_module.cmake

```diff
@@ -120,6 +120,13 @@ function(iree_bytecode_module)
     list(APPEND _ARGS "-iree-llvm-link-embedded=false")
   endif()
 
+  # Support testing in TSan build dirs. Unlike other sanitizers, TSan is an
+  # ABI break: when the host code is built with TSan, the module must be too,
+  # otherwise we get crashes calling module code.
+  if (IREE_BYTECODE_MODULE_ENABLE_TSAN)
```

Comment:
CMake style nit: we don't put a space between `if` and `(`

Reply:
Done